draft-ietf-masque-connect-ip.md

```diff
@@ -967,6 +969,97 @@ route is restricted to 192.0.2.0/24, rather than 0.0.0.0/0.
 ~~~
 {: #fig-split-tunnel title="VPN Split-Tunnel Capsule Example"}
 
+## Site-to-Site VPN
+
+The following example shows how to connect a branch office network to a
+corporate network such that all machines on those networks can communicate.
+In this example, the IP proxying client is attached to the branch office
+network 192.0.2.0/24, and the IP proxy is attached to the corporate network
+233.252.0.0/24. There are legacy clients on the branch office network that
```

Comment:
I would recommend that you use 203.0.113.0/24 which is a unicast test net, rather than the multicast testnet currently chosen.

Reply:
Good catch! That's the prefix I wanted to use and must have copy-pasted the next line. Fixed